### PR TITLE
Add comment when linking task

### PR DESCRIPTION
**Pull request by zpawn**
Story https://app.shortcut.com/deskpro/story/163697/wrike-new-features

### CODE REVIEW

File: jest.setup.tsx [hunk 1]
```diff
@@ -28,6 +28,12 @@ global.React = React;
 //@ts-ignore
 global.ResizeObserver = ResizeObserver;
 
+let currentContext: any = mockTicketContext;
```
Comment (HappyPaul55) [inline jest.setup.tsx:31]: Don't use `any`

File: jest.setup.tsx [hunk 2]
```diff
@@ -28,6 +28,12 @@ global.React = React;
...
+
+export const setCurrentContext = (context: any) => {
```
Comment (HappyPaul55) [inline jest.setup.tsx:33]: Don't use any

File: src/hooks/useLinkedNote.ts [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+import type { ITaskFromList, INote } from "@/services/wrike/types";
+
+export type Result = {
+  isLoading: boolean,
+  addLinkNote: (task: ITaskFromList|TaskType) => Promise<INote|null>,
+  addUnlinkNote: (task: ITaskFromList|TaskType) => Promise<INote|null>,
+};
+
+const getHTMLLink = (link: string) => {
+  return `<a href="${link}">${link}</a>`;
```
Comment (HappyPaul55) [inline src/hooks/useLinkedNote.ts:14]: Why HTML as text instead of ReactNode

**Review by HappyPaul55 — CHANGES_REQUESTED** [review]

File: src/hooks/useLinkedNote.ts (hunk 3 above)
Comment (zpawn) [reply src/hooks/useLinkedNote.ts:14]: Notes in Wrike are stored as HTML strings.

File: jest.setup.tsx (hunk 1 above)
Comment (zpawn) [reply jest.setup.tsx]: fixed

File: jest.setup.tsx (hunk 2 above)
Comment (zpawn) [reply jest.setup.tsx]: fixed

**Review by HappyPaul55 — APPROVED** [review]

**Pull request merged**